stable/awsebscsiprovisioner/values.yaml

```
@@ -26,32 +26,31 @@ registrar:
       repository: "quay.io/k8scsi/csi-node-driver-registrar"
       tag: "v1.1.0"
 
-
 provisioner:
   # True if enable volume scheduling for dynamic volume provisioning
   enableVolumeScheduling: false
   image:
     repository: "quay.io/k8scsi/csi-provisioner"
-    tag: "v1.3.0"
+    tag: "v1.4.0"
 
 attacher:
   image:
     repository: "quay.io/k8scsi/csi-attacher"
-    tag: "v1.2.0"
+    tag: "v1.2.1"
```

```suggestion
    tag: "v2.0.0"
```

support wise we should be able to go https://kubernetes-csi.github.io/docs/external-attacher.html